### PR TITLE
feat: adds reusable workflow for hasura projects

### PR DESCRIPTION
**Pull request by fitimbyttyqi**
[issues-30/hasura-reusable-actions](https://github.com/ornio-no/issues/issues/30)

### CODE REVIEW

**Comment by fitimbyttyqi** [comment]
@oxodesign Hasura setups are a bit different from others like dockerfiles, they don't exists.
I am not actually sure how hasura projects are setup using docker but this is what I came up with.
Lemme know if we can further improve it

File: .github/workflows/hasura-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Hasura Build and Deploy
+
+on:
+  workflow_call:
+    inputs:
+      environment:
```
Comment (oxodesign) [inline .github/workflows/hasura-deploy.yml:6]: @fitimbyttyqi just a small change, i used just "env" on k8s maybe we should align this, either use "environment" or "env"

Comment (fitimbyttyqi) [reply .github/workflows/hasura-deploy.yml:6]: @oxodesign It's up to you, i like environment more but lemme know I can quickly fix it

**Comment by fitimbyttyqi** [comment]
@oxodesign Checkout the new changes a pushed a bit ago

**Review by oxodesign — APPROVED** [review]

**Pull request merged**